docs/koji.md

```diff
@@ -0,0 +1,299 @@
+# Koji
+
+[Koji](https://pagure.io/koji/) is an RPM build and tracking system. The Foreman project Koji instance lives at [Foreman Koji](http://koji.katello.org/koji/).
+
+## Koji Instance
+
+The Koji instance is running in AWS EC2 (us-east-1).
+
+  * i3.xlarge instance (4 CPUs, 32 GB RAM, 900 GB SSD NVMe).
```

Comment:
Should this be formated similar to what we did in https://github.com/theforeman/foreman-infra/pull/1421?